config/dependency.go

```diff
@@ -136,6 +217,23 @@ func getTerragruntOutput(dependencyConfig Dependency, terragruntOptions *options
 	return &convertedOutput, errors.WithStackTrace(err)
 }
 
+// runTerragruntShow uses terragrunt running functions to show the current state of the target config.
+func runTerragruntShow(terragruntOptions *options.TerragruntOptions, targetConfig string) (string, error) {
+	var stdoutBuffer bytes.Buffer
+	stdoutBufferWriter := bufio.NewWriter(&stdoutBuffer)
+	targetOptions := terragruntOptions.Clone(targetConfig)
+	targetOptions.TerraformCliArgs = []string{"show", "-no-color"}
+	targetOptions.Writer = stdoutBufferWriter
+	err := targetOptions.RunTerragrunt(targetOptions)
+	if err != nil {
+		return "", errors.WithStackTrace(err)
+	}
+
+	stdoutBufferWriter.Flush()
+	outputString := stdoutBuffer.String()
+	return outputString, nil
```

Comment:
Do we need a wrapper method for executing Terragrunt commands in a child module and handling all the stdout/stderr stuff? I feel like I've seen this large block of code before?

Reply:
Yes, but actually once I removed `show` it is no longer necessary, so I will keep the one output call block for now.